morphia/src/main/java/org/mongodb/morphia/MorphiaUtils.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mongodb.morphia;
```

Comment:
To make it more obvious that its internal code and that any changes aren't considered binary breaking perhaps you could namespace it?  eg: `org.mongodb.morphia.internal`.  This is the approach taken in Java driver.